packages/platform/atoms/connect/conferencing-apps/ConferencingAppsViewWebWrapper.tsx

```diff
@@ -0,0 +1,187 @@
+"use client";
+
+import { useReducer } from "react";
+
+import DisconnectIntegrationModal from "@calcom/features/apps/components/DisconnectIntegrationModal";
+import SettingsHeader from "@calcom/features/settings/appDir/SettingsHeader";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { QueryCell } from "@calcom/trpc/components/QueryCell";
+import { trpc } from "@calcom/trpc/react";
+import { Button, EmptyScreen, showToast, SkeletonContainer, SkeletonText } from "@calcom/ui";
+import { AppList } from "@calcom/web/components/apps/AppList";
```

Comment:
move this component in the features package, packages should not import from apps, only the other way round